integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -229,7 +230,14 @@ public int getattr(String path, FileStat stat) {
   private int getattrInternal(String path, FileStat stat) {
     final AlluxioURI uri = mPathResolverCache.getUnchecked(path);
     try {
-      URIStatus status = mFileSystem.getStatus(uri);
+      URIStatus status = null;
+      // Handle special metadata cache operation
+      if (mConf.getBoolean(PropertyKey.USER_CLIENT_SPECIAL_COMMAND_ENABLED)
+          && mConf.getBoolean(PropertyKey.USER_METADATA_CACHE_ENABLED)) {
```

Comment:
If Fuse special command enabled & path starts with `".alluxiocli"`
-> enter a new method to process the path and get subcommand
-> if subcommand is metadata cache -> then go to new method to process metadata cache related commands